packages/@aws-cdk/aws-efs/lib/access-point.ts

```diff
@@ -0,0 +1,131 @@
+import { Construct, IResource, Resource, Stack } from '@aws-cdk/core';
+import { IFileSystem } from './efs-file-system';
+import { CfnAccessPoint } from './efs.generated';
+
+/**
+ * Interface to implement AccessPoint
+ */
+export interface IAccessPoint extends IResource {
+  /**
+   * The ID of the AccessPoint
+   *
+   * @attribute
+   */
+  readonly accessPointId: string;
+}
+
+/**
+ * Properties for the AccessPoint
+ */
+export interface AccessPointProps {
+  /**
+   * The efs filesystem
+   */
+  readonly filesystem: IFileSystem;
+
+  /**
+   * Specifies the POSIX user ID to apply to the RootDirectory. Accepts values from 0 to 2^32 (4294967295).
+   *
+   * @default 1000
+   */
+  readonly ownerUid?: string;
+
+  /**
+   * Specifies the POSIX group ID to apply to the RootDirectory. Accepts values from 0 to 2^32 (4294967295).
+   *
+   * @default 1000
+   */
+  readonly ownerGid?: string;
+
+  /**
+   * Specifies the POSIX permissions to apply to the RootDirectory, in the format of an octal number representing
+   * the file's mode bits.
+   *
+   * @default 755
+   */
+  readonly permissions?: string;
+
+  /**
+   * Specifies the path on the EFS file system to expose as the root directory to NFS clients using the access point
+   * to access the EFS file system
+   *
+   * @default /export
+   */
+  readonly path?: string;
+
+  /**
+   * The POSIX user ID used for all file system operations using this access point.
+   *
+   * @default 1000
```

Comment:
Instead of defaulting to `1000`, can we make this default to whatever `ownerUid` is?

And same for `posixUserGid`.

Reply:
Agree!!!